src/ov_utils.cpp

```diff
@@ -64,4 +64,46 @@ Status blobClone(InferenceEngine::Blob::Ptr& destinationBlob, const InferenceEng
     return StatusCode::OK;
 }
 
+Status constBlobClone(InferenceEngine::Blob::Ptr& destinationBlob, const InferenceEngine::Blob::CPtr sourceBlob) {
```

Comment:
We could make the second parameter templated, since both implementations are the same?